docs/collecting-data/collecting-from-own-applications/react-native-tracker/expo/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Support for Expo"
+date: "2022-02-03"
+sidebar_position: 35
+---
+
+# Support for the Expo framework
+
+The React Native tracker supports the [Expo framework](https://expo.dev/). However, the support is different for the bare and the managed workflow.
+
+You may install the tracker using:
+
+```sh
+npx expo install @snowplow/react-native-tracker
+```
+
+## Expo bare workflow
+
+The tracker works without limitations on the bare workflow. Since the tracker is able to use iOS and Android native code using the mobile trackers, it provides the full functionality of the React Native tracker.
```

Comment:
```suggestion
The tracker works without limitations on the bare workflow. Since the tracker is able to run iOS and Android native code using the mobile trackers, it provides the full functionality of the React Native tracker.
```